projects/MotionLayoutVerification/app/src/main/res/layout/verification_044.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
These XML files don't seem to do anything.

Reply:
removed